support-frontend/assets/pages/contributions-landing/components/ContributionFormFields.jsx

```diff
@@ -85,7 +85,7 @@ function withProps(props: PropTypes) {
     lastName,
     email,
     isSignedIn,
-    state,
+    billingState,
```

Comment:
This is really helpful! So confusing otherwise.

Reply:
agreed!